migration_steps/shared/mapping_definitions/tables/table_definitions.json

```diff
@@ -137,7 +137,15 @@
         "destination_table_name": "persons",
         "entity_name": "clients",
         "required_entities": "",
-        "source_conditions": "",
+        "source_conditions": {
```

Comment:
any reason you went for `source_conditions` on the table instead of `requires_transformation` on the columns?

Reply:
Well transforms didn't handle multiple chained transforms but really I didn't really understand the difference between the two. After some soul searching I guess conditions should be more of a filter and transforms actual transforms so I've rewritten as transform and made it handle multiple chained transforms.